docs/source/io_formats/tallies.rst

```diff
@@ -154,14 +154,30 @@ For each filter type, the following table describes what the ``bins`` attribute
 should be set to:
 
 :cell:
-  A list of unique IDs for cells in which the tally should be accumulated.
+  A list of cells or unique IDs for cells in which the tally should be 
```

Comment:
Since this is describing XML, the original wording was correct, i.e., it should be a list of unique IDs for cells.